### ISSUE TITLE
Place exact matches at the top of list

### ISSUE THREAD

**Issue by roaldarbol** [sneakers-the-rat/journal-rss#13]
We should try to place exact hits at the top. Currently it would be impossible to follow Science as they are too far down the list. 
I don't know whether there are other situations where we should think about the list order - I can't think of any right now.

**Comment by sneakers-the-rat** [comment]
Yep noticed this with Cell too. At the moment we're just taking crossref's search results and printing them, so yeah checking for an exact title match and promoting it would definitely be one easy thing to do here.

Their tips page ( https://www.crossref.org/documentation/retrieve-metadata/rest-api/tips-for-using-the-crossref-rest-api/ ) says not to overly constrain the query, but we might want to think about not trying to do pure single bar search and instead have a field selector (ie. A dropdown to select explicit "title" search) which I think will make it easier to do different kinds of feeds like keywords. I bet we get better search results if we specify the query is a title query vs. A general query.

The other thing to do is increase the number of results or do paging on scroll, which would be nice to have ofc

**Comment by roaldarbol** [comment]
Yup, sounds good to me. I did find a search-bar like that (but with JS) 😅.
For table, we could just have a multipage table, or have a max-height, but where it's possible to scroll within the table container (I prefer pages I think).
